azure-fs-aks/Program.fs

```diff
@@ -0,0 +1,131 @@
+﻿module Program
+
+open Pulumi.FSharp
+open Pulumi.Azure.Core
+open Pulumi.Azure.ContainerService
+open Pulumi.Azure.ContainerService.Inputs
+open Pulumi.Azure.AD
+open Pulumi.Azure.Network
+open Pulumi.Random
+open Pulumi.Tls
+
+[<RequireQualifiedAccess>]
+module Pulumi =
```

Comment:
I would choose a different name for this module to avoid confusing with Pulumi libraries

Reply:
I can do that to avoid confusion, the thinking behind the naming is basically that the functions are sort of extension functions to `Pulumi`. Will change to `Helpers`.